src/util/interpolate.test.ts

```diff
@@ -117,6 +118,16 @@ describe('interpolate', () => {
         expect(i11nFn(0.5)).toEqual(new Padding([0.5, 1, 3, 2]));
     });
 
+    test('interpolate projection', () => {
+        
+        const Globe = new Projection('globe');
+        const Mercator = new Projection('mercator');
+
+        const i11nFn = (t: number) => interpolate.projection(Globe, Mercator, t);
+        expect(i11nFn(0.5)).toBeInstanceOf(ProjectionTransition);
+        expect(i11nFn(0.5).toString()).toBe('["globe", "mercator", 0.5]');
```

Comment:
This what got me confused, I would advise to compare using "toEqual" against an object without the toString conversation.

Reply:
You can check the babies of from, to etc separately.